src/components/SelectedPokemon/SelectedPokemon.tsx

```diff
@@ -23,10 +57,20 @@ class SelectedPokemonPure extends React.Component<Props> {
                     <PokemonSkills>
                       {`id: ${id} height: ${height} weight: ${weight}`}
                     </PokemonSkills>
+                    <PokemonStarWrapper onClick={() => this.handleStarClick(id, name, height, weight)}>
+                      <Image src={(isFavorite) ? `/img/star-fav.png` : `/img/star-empty.png`} />
```

Comment:
tutaj raczej nie powinieneś wgl renderowac elementu `<Image` kiedy nie jest `isFavorite`, a jeśli coś się przesuwa to ustawić to dobrze w css'ie